hoprd/db/entity/Cargo.toml

```diff
@@ -22,6 +22,7 @@ sea-orm-migration = { workspace = true }
 [dependencies]
 sea-orm = { workspace = true }
 thiserror = { workspace = true }
+serde = { workspace = true }
```

Comment:
It is a good practice to hide `serde` behind a feature with the same name.
Then you'd hide the `#[derive(Serialize, Deserialize)]` on your types behind `#[cfg_attr(feature = "serde", Serialize)]`